libs/hdf-converters/src/zap-mapper.ts

```diff
@@ -140,26 +133,42 @@ export class ZapMapper extends BaseConverter {
               riskdesc: {path: 'riskdesc'},
               check: {transformer: checkText}
             },
-            descriptions: [],
             refs: [],
             source_location: {},
-            code: '',
+            title: {path: 'name'},
+            id: {path: 'pluginid'},
+            desc: {path: 'desc', transformer: parseHtml},
+            impact: {path: 'riskcode', transformer: impactMapping},
+            code: {
+              transformer: (vulnerability: Record<string, unknown>): string => {
+                return JSON.stringify(vulnerability, null, 2);
+              }
+            },
             results: [
               {
                 path: 'instances',
                 status: ExecJSON.ControlResultStatus.Failed,
                 code_desc: {transformer: formatCodeDesc},
-                run_time: 0,
                 start_time: {path: '$.@generated'}
               }
             ]
           }
         ],
         sha256: ''
       }
-    ]
+    ],
+    passthrough: {
+      transformer: (data: Record<string, any>): Record<string, unknown> => {
```

Comment:
Avoid uses of "any" (I think this has been commented on other PR's already though)